lib/rialto/etl/service_client/retriable_connection_factory.rb

```diff
@@ -10,29 +10,40 @@ module ServiceClient
       class RetriableConnectionFactory
         DEFAULT_MAX_RETRIES = 6
         DEFAULT_MAX_INTERVAL = Float::MAX
-        class_attribute :logger
+        class_attribute :logger, :max_retries, :max_interval
 
-        # rubocop:disable Metrics/MethodLength
-        def self.build(uri:, headers:, logger: default_logger, max_retries: nil, max_interval: nil)
+        # rubocop:disable Metrics/ParameterLists
+        def self.build(uri:, headers: nil, oauth_token: nil, logger: default_logger, max_retries: nil, max_interval: nil)
```

Comment:
Do we want to allow invocations of `.build` without the `headers:` arg now?

Reply:
Good question, the existing OAUTH version doesn't pass any headers... and setting them to `nil` didn't seem to cause issue... but I can review.